tgui/packages/tgui/interfaces/TransceptionInterlink/type.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author glowbold (https://github.com/pgmzeta)
+ * @license MIT
+ */
+
+export interface TransceptionInterlinkData {
+  pads: PadData[],
+  crate_count: number
+}
+
+export interface PadData {
+  device_index: string, // key in the interlink's manifest list
```

Comment:
Is this actually a string?

Reply:
yes; updated the variable name to keep it consistent with other usage